packages/bot-web-ui/src/components/quick-strategy/selects/__tests__/contract_type.spec.tsx

```diff
@@ -15,6 +15,22 @@ jest.mock('@deriv/bot-skeleton/src/scratch/blockly', () => jest.fn());
 jest.mock('@deriv/bot-skeleton/src/scratch/dbot', () => jest.fn());
 jest.mock('@deriv/bot-skeleton/src/scratch/hooks/block_svg', () => jest.fn());
 
+jest.mock('formik', () => ({
+    ...jest.requireActual('formik'),
+    Field: jest.fn(({ children }) =>
+        children({
+            field: {
+                value: 'CALL',
+            },
+        })
+    ),
+    useFormikContext: jest.fn(() => ({
+        setFieldValue: jest.fn(),
+        validateForm: jest.fn(),
+        values: { symbol: '1HZ100V', tradetype: 'callput', type: 'CALL' },
+    })),
+}));
+
```

Comment:
@maryia-matskevich-deriv is it necessary to mock `formik` here? I think we can just wrap the test component with `<Formik />` to set the initial values from the test file itself and let the real component work in regular flow?

Reply:
Up to the current point, the form values were undefined, I had to mock them. But now I see the test passes without this mock, so I remove it.

Reply:
unfortunately I've returned the changes, we need the mock